modules/release-notes/ref-release-notes-breaking-changes.adoc

```diff
@@ -4,6 +4,59 @@
 
 This section lists breaking changes in {product} {product-version}.
 
+[id="removed-functionality-rhidp-6215"]
+== The Topology-specific permission `topology.view.read` is removed
+
+Previously, the Topology plugin used `topology.view.read` permission to control access. Users were unable to configure Topology permissions using the RBAC UI. With this update, users can configure Kubernetes plugin permissions using the RBAC UI, which now governs the access to the Topology plugin. You can now use Kubernetes plugin permissions `kubernetes.clusters.read` and `kubernetes.resources.read` for the Topology plugin, as the topology-specific permission `topology.view.read` is removed. 
+
+If you are using a CSV permission file, update the following line:
+
+.Old topology permission definition
+[source,csv]
+----
+p, role:default/topology-viewer, topology.view.read, read, allow
+----
+
+.New topology permission definition
+[source,csv]
+----
+p, role:default/topology-viewer, kubernetes.clusters.read, read, allow
+p, role:default/topology-viewer, kubernetes.resources.read, read, allow
+----
+
+
+.Additional resources
+* link:https://issues.redhat.com/browse/RHIDP-6215[RHIDP-6215]
+
+[id="removed-functionality-rhidp-7365"]
+== Migration to the core Auditor service
+
+The Auditor format, including audit fields and event names, and IDs, has been updated to align with the new Auditor service conventions. Filtering queries based on the old format may fail to work as expected.
+
+
+.Additional resources
+* link:https://issues.redhat.com/browse/RHIDP-7365[RHIDP-7365]
+
+[id="removed-functionality-rhidp-7373"]
+== {product} introduces the Backstage Audit Log Service
+
+{product} {product-version} introduces the Backstage Audit Log Service, which replaces the custom audit logging system. This a significant structural and behavioral change to how audit events are generated and consumed.

```

Comment:
```suggestion
{product} {product-version} introduces the Backstage Audit Log Service, which replaces the custom audit logging system. This is a significant structural and behavioral change to how audit events are generated and consumed.

```

Reply:
reminder that the RN script will apply typo corrections when you run the script again after updating the JIRA RN text fields. So no need to comment here -- just fix the typos in https://issues.redhat.com/browse/RHIDP-7373 directly.